test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonLight/SingletonBatchRoundtripJsonLightTests.cs

```diff
@@ -1256,6 +1309,81 @@ private void BatchJsonLightTestUsingBatchFormat(BatchFormat batchFormat, int idx
             this.ClientReadSingletonBatchResponse(responsePayload, GetContentTypeHeader(batchFormat));
         }
 
+        private void BatchJsonLightTestUsingBatchFormat_UsingStreamCopy(BatchFormat batchFormat, int idx, Action<IContainerBuilder> configureAction = null)
+        {
+            byte[] requestPayload = null;
+            switch (idx)
+            {
+                case 0:
+                    requestPayload = ClientWriteSingletonBatchRequest(
+                        GetContentTypeHeader(batchFormat));
+                    break;
+
+                case 1:
+                    requestPayload = CreateBatchRequestWithChangesetFirstAndQueryLast(
+                        GetContentTypeHeader(batchFormat));
+                    break;
+
+                default:
+                    throw new ArgumentException("Unknown batch request index value", "idx");
+            }
+
+            VerifyPayload(requestPayload, batchFormat, true /*for request*/, idx);

```

Comment:
Maybe use `<parameterName>: true` instead of `true /*for request*/`

Including other statements below